models/quality_measures/intermediate/nqf_0097_medication_reconciliation_post_discharge/quality_measures__int_nqf0097_denominator.sql

```diff
@@ -0,0 +1,193 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with visit_codes as (
+
+    select
+          concept_name
+        , code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+
+)
+
+, valid_concepts as (
```

Comment:
We need to list out the appropriate concepts here since this table includes exclusion and numerator concepts.